src/stylesheets/elements/_table.scss

```diff
@@ -2,29 +2,30 @@ table {
   border-spacing: 0;
   margin: 2em 0;
   min-width: 100%;
+}
 
-  thead {
-    tr {
-      background-color: $color-gray-lightest;
-    }
-
-    th {
-      text-align: left;
-    }
+thead {
```

Comment:
This is the only SelectorDepth I tried to fix, by moving the `thead` definition outside of the `table`. Again, let me know if that breaks anything.